ui/text_displayer.js

```diff
@@ -168,6 +168,133 @@ shaka.ui.TextDisplayer = class {
     }
   }
 
+  /**
+   * Returns a safe collection of html elements for display as a single cue.
+   * @param {string} cueBody
+   * @return {Array<HTMLElement>}
```

Comment:
This never returns null, so this should be a non-nullable Array, prefixed with an exclamation point (!Array<HTMLElement>)

Reply:
Neat. This has been applied to the new method structure.